src/utils/data_tables.ts

```diff
@@ -0,0 +1,8 @@
+import * as t from "io-ts";
+
+export const TableEntity = t.type({
```

Comment:
see above comment

Reply:
Resolved in [491818b](https://github.com/pagopa/io-functions-commons/pull/384/commits/491818b5019eaf91be790915b87b44909b57785e)